sbin/cubeit

```diff
@@ -500,6 +521,7 @@ IFS=$OLDIFS
 	    # create a configuration that can be read by the cubeit-installer
 	    if [ -v TARGET_CONFIG_FILE -a -n "$TARGET_CONFIG_FILE" ]; then
 		cat $TARGET_CONFIG_FILE |
+			sed "s|ARTIFACTS_DIR=.*|ARTIFACTS_DIR=${INSTALLER_TARGET_IMAGES_DIR}|" | \
```

Comment:
it isn't clear why this is being changed. We are already modifying ARTIFACTS_DIR, and now we are modfying it for ARTIFACTS_DIR=.